openmetadata-service/src/main/java/org/openmetadata/service/governance/workflows/WorkflowHandler.java

```diff
@@ -0,0 +1,237 @@
+package org.openmetadata.service.governance.workflows;
+
+import static org.openmetadata.service.governance.workflows.elements.TriggerFactory.getTriggerWorkflowId;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.flowable.bpmn.converter.BpmnXMLConverter;
+import org.flowable.common.engine.api.FlowableException;
+import org.flowable.common.engine.api.FlowableObjectNotFoundException;
+import org.flowable.engine.HistoryService;
+import org.flowable.engine.ProcessEngine;
+import org.flowable.engine.ProcessEngineConfiguration;
+import org.flowable.engine.RepositoryService;
+import org.flowable.engine.RuntimeService;
+import org.flowable.engine.TaskService;
+import org.flowable.engine.history.HistoricProcessInstance;
+import org.flowable.engine.impl.cfg.StandaloneProcessEngineConfiguration;
+import org.flowable.engine.repository.ProcessDefinition;
+import org.flowable.engine.runtime.Execution;
+import org.flowable.engine.runtime.ProcessInstance;
+import org.flowable.task.api.Task;
+import org.openmetadata.service.OpenMetadataApplicationConfig;
+import org.openmetadata.service.exception.UnhandledServerException;
+import org.openmetadata.service.jdbi3.locator.ConnectionType;
+
+@Slf4j
+public class WorkflowHandler {
+  private final RepositoryService repositoryService;
+  private final RuntimeService runtimeService;
+  private final TaskService taskService;
+  private final HistoryService historyService;
+  private static WorkflowHandler instance;
+  private static volatile boolean initialized = false;
+
+  private WorkflowHandler(OpenMetadataApplicationConfig config) {
+    ProcessEngineConfiguration processEngineConfiguration =
+        new StandaloneProcessEngineConfiguration()
```

Comment:
related to the above, maybe this object has some migration utilities to leverage

Reply:
Not sure I understood the comment for this specific configuration